concent_api/core/models.py

```diff
@@ -35,9 +51,21 @@ class ReceiveOutOfBandStatus(Model):
     timestamp   = DateTimeField()
     delivered   = BooleanField(default = False)
 
+    objects = ReceiveStatusManager()
+
     def __str__(self):
         return 'ReceiveOutOfBandStatus #{}, message:{}'.format(self.id, self.message)
 
     class Meta:
         verbose_name        = ('ReceiveOutOfBand status')
         verbose_name_plural = ('ReceiveOutOfBand statuses')
+
+
+class MessageAuth(Model):
+    """
+    This class is used to store provider and requestor keys for message exchange related to given initial message.
+    """
+
+    message              = OneToOneField('Message', related_name='auth')
+    provider_public_key  = Base64Field()
+    requestor_public_key = Base64Field()
```

Comment:
We should have validations for these fields. The fields should ensure that they get base64-encoded data. Unless you coded it so that you assign binary data to them - in that case the fields should only accept `bytes`.

Reply:
Please check updated code and decide if this comment is still valid.